webapp/Dockerfile-qe

```diff
@@ -26,8 +26,9 @@ ENV COVERAGE_FILE='/tmp/.coverage'
 
 EXPOSE 8080
 
-ADD /webapp/*.sh     /webapp/
-ADD /webapp/*.py     /webapp/
-ADD /common/*.py     /webapp/common/
```

Comment:
please add back `ADD /common/*.py     /webapp/common/`
Jenkins is failing because of
```
Everything is up - executing command
Traceback (most recent call last):
  File "webapp/app.py", line 18, in <module>
    from cache import Cache
  File "/webapp/cache.py", line 9, in <module>
    from common.logging_utils import get_logger
ModuleNotFoundError: No module named 'common'
```

Reply:
Fixed, must have been merge error.